Makefile

```diff
@@ -18,9 +18,9 @@ install: check-prerequisites requirements build
 
 .PHONY: build
 build: clean
-	python -m linodecli bake ${SPEC} --skip-config
+#	python -m linodecli bake ${SPEC} --skip-config
 	python3 -m linodecli bake ${SPEC} --skip-config
-	cp data-2 linodecli/
+#	cp data-2 linodecli/
```

Comment:
Was this change intentionally included in this PR?  [It's probably safe to drop python2 support now](https://pypistats.org/packages/linode-cli), but I would figure doing so would warrant its own PR with all the necessary changes.